src/scripts/types/types.ts

```diff
@@ -1,5 +1,5 @@
 export interface ResourceCodeInfo {
-  primaryCodePath: string;
```

Comment:
We should double check that where this is used can handle the change to this being an optional field. Might be just a quick change on when we do version bump though.

Reply:
Yes- I already made the change in fqm-testify. I don't know of this file being used elsewhere at the moment, but as we update fhir-spec-tools elsewhere, we should keep our eyes out.